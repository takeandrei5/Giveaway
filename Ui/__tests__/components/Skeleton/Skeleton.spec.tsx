import { Skeleton } from '@components';
import { render } from '@testing-library/react';

describe('Skeleton', () => {
	it('should match snapshot', () => {
		// Act
		const { container } = render(<Skeleton />);

		// Assert
		expect(container).toMatchSnapshot();
	});
});
