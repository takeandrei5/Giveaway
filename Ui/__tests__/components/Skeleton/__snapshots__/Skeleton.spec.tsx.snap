// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should match snapshot 1`] = `
<div>
  <div
    class="chakra-skeleton css-1twknw3"
  />
</div>
`;
